projects/components/src/quick-search/quick-search.provider.ts

```diff
@@ -37,5 +45,17 @@ export abstract class QuickSearchProviderDefaults implements QuickSearchProvider
     sectionName = '';
     order = -1;
     data: unknown;
+    abstract shouldDebounceInput: boolean;
```

Comment:
Why abstract? Shouldn't it just give it a default value like the other settings?